pages/desktop/details.py

```diff
@@ -551,10 +552,9 @@ class ContributionSnippet(Page):
 
         def __init__(self, base_url, selenium):
             Page.__init__(self, base_url, selenium)
-
-            WebDriverWait(self.selenium, self.timeout).until(
-                lambda s: s.find_element(*self._make_contribution_button_locator).is_displayed(),
-                "Timeout waiting for 'make contribution' button.")
+            self.wait.until(expected.visibility_of_element_located(
```

Comment:
I assume this is just code cleanup/improvement. It doesn't look the the functionality has changed with your version.